### ISSUE TITLE
Problems with space between items.

### ISSUE THREAD

**Issue by davidmarinangeli** [xmuSistone/VegaLayoutManager#26]
Hello, thanks for this spectacular library. 

I have a problem with my RecyclerView: the linear layout manager works fine but as soon as I try yours, I have this problem (Photos: [https://photos.app.goo.gl/KB84ASKxTwdrZkVH2](https://photos.app.goo.gl/KB84ASKxTwdrZkVH2) )

Is there a way to fix it? 

As you can see, there is A LOT of space between some items and the last doesn't have the little textview in the bottom of the card. 

Thanks

**Comment by xmuSistone** [comment]
yeah, it's a bug. in your code, r u using decoration to make the space?

**Comment by davidmarinangeli** [comment]
Nope, here is item's XML: [https://github.com/davidmarinangeli/ErmesAndroid/blob/master/app/src/main/res/layout/default_sport_event_item.xml](url)

and the Recylcerview is set using:

`adapter = new MainAdapter(getContext());
        recyclerView = rootView.findViewById(R.id.main_contenitore);
        recyclerView.setLayoutManager(new VegaLayoutManager());
        recyclerView.setAdapter(adapter);
        recyclerView.setNestedScrollingEnabled(false);`
